### PR TITLE
Compatibility with Symfony3

### PR DESCRIPTION
**Pull request by vpx**

### CODE REVIEW

**Comment by oanhnn** [comment]
You should using file deployer.phar and shouldn't require deployer in composer.json.

**Comment by elfet** [comment]
@oanhnn a lot of people install it via composer. I think it's good to have both ways.

File: composer.json [hunk 1]
```diff
@@ -23,12 +23,12 @@
         "bin/dep"
     ],
     "require": {
-        "php": ">=5.4.0",
+        "php": ">=5.5.9",
```
Comment (elfet) [inline composer.json]: 5.4.0 still supported.

File: composer.json [hunk 2]
```diff
@@ -23,12 +23,12 @@
...
         "elfet/pure": "~1.1",
```
Comment (elfet) [inline composer.json]: pure need to support it too.

**Comment by cedricziel** [comment]
This pull request is obsolete since i cherry-picked the relevant commit which was merged. Thank you @vpx !

**Pull request closed without merging**